backend/app.js

```diff
@@ -3,9 +3,14 @@ const express = require('express');
 const mongoose = require('mongoose');
 const config = require('./config/config');
 const userRoutes = require('./routes/userRoutes');
-
 const app = express();
 
+// Start the server
+app.listen(3000, () => {
```

Comment:
Hi Mahmoud. I was just wondering that if we start the server at localhost 3000 and test the front end at the same time, would that be a problem? Like we have to change the ports of the client side somehow? Please correct me if my concern is not necessary. Thanks